python/paddle/fluid/framework.py

```diff
@@ -2858,6 +2858,12 @@ def create_parameter(self, *args, **kwargs):
             param = ParamBase(*args, **kwargs)
         else:
             param = Parameter(global_block, *args, **kwargs)
+            # NOTE: Why only set stop_gradient=False in static mode
+            # Because in dygraph mode, the `stop_gradient` and 
+            # `trainable` are related, if we set `stop_gradient` 
```

Comment:
Because in dygraph mode, the `stop_gradient` and `trainable` are related, if we **not** set `stop_gradient` here, the user given `trainable` argument will not take effect.

Or, may be can:

Because in dygraph mode, the `stop_gradient` and `trainable` are related, and `trainable` is specified by user. There is no need to set `stop_gradient` for ParamBase.

Reply:
done, thx